packages/zoe/src/contractSupport/bondingCurves.js

```diff
@@ -41,10 +38,10 @@ export const getInputPrice = (
   assert(inputReserve > 0, X`inputReserve ${inputReserve} must be positive`);
   assert(outputReserve > 0, X`outputReserve ${outputReserve} must be positive`);
 
-  const oneMinusFeeScaled = BIG_10000 - BigInt(feeBasisPoints);
+  const oneMinusFeeScaled = 10_000n - BigInt(feeBasisPoints);
```

Comment:
Please rename to BASIS_POINTS rather than using a manifest constant.

Reply:
yes, actual abstraction that says more than the number! Done.